c2corg_ui/templates/helpers/view.html

```diff
@@ -315,11 +316,24 @@ <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($even
 
   % else :
     % if text in locale and locale[text]:
+    <%
+      openTab =  text == 'route_description' or text == 'hut_comment'
+    %>
       <div class="view-details-info col-xs-12">
-        <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#${text}">
+        <h3 class="heading
+            % if not openTab:
+              closed
+            % endif
+            show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#${text}">
```

Comment:
I prefer to use ${'' if openTab else 'closed'} or ${'closed' if not openTab} to avoid carriage return inside class attribute value.
I prefer to put carriage return between properties, each properties on one line, for lisibility.

It is common to use underscore but not uppercase in python variable names ex: open_tab